src/Contao/Widgets/MultiColumnWizard.php

```diff
@@ -772,10 +772,6 @@ public function generate($overwriteRowCurrentRow = null, $onlyRows = false)
 
                 $arrTinyMCE[] = $strKey;
             }
-
-            if ($arrField['inputType'] == 'hidden') {
```

Comment:
is this change related to the array key warning?

Reply:
Do you mean, deleting instead of null-check? This block obviously doesn't do anything. I checked a few levels of changes using `blame`, and it seems this check is a forgotten leftover, that lost its purpose after refactoring. It was not previously at the end of `foreach` cycle.

Since it does contribute to the title problem, I think yes, it is related.